tests/test_aiml_skill.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from pathlib import Path
```

Comment:
PEP 8 imports order

Reply:
Hm, don't catch idea.
```Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```
Is this comment related to order of imports or for lack of blank lines?

Reply:
```
from pathlib import Path
import pytest
```

Is this right order?

Reply:
`pytest` is not in a standard library